docs/api/rpc-params.md

```diff
@@ -0,0 +1,158 @@
+- `finality`: (string) filter for status of block finalization [ `optmistic` | `near-final` | `final` ] 
+
+
+*At time of writing, `finality` is only available on StageNet.*
+
+To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.nearprotocol.com/status
+
+### Using block height
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using block hash
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list", 
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using `optimistic` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "optimistic"
+             }'
+```
+
+### Using `near-final` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "near-final"
+             }'
+```
+
+### Using `final` finality
+
+```bash
+http post https://staging-rpc.nearprotocol.com jsonrpc=2.0 id=dontcare \
+    method=query \
+    'params:={
+                "request_type": "view_access_key_list",
+                "account_id": "node0", 
+                "block_id": 33442,
+                "finality": "final"
+             }'
+```
+
+<blockquote class="info">
+<strong>did you know?</strong><br><br>
+
+The `request_type` parameter can take any one of the following values (and associated parameters)
+
+```rust
+view_account          account_id: string
+view_state            account_id: string, prefix: string /* base64 */
+view_access_key       account_id: string, public_key: string
+view_access_key_list  account_id: string
+call_function         account_id: string, method_name: string, args: string /* base64 */
+```
+
+For a concrete example we can use the NEAR StageNet environment.
+
+1. The first two examples above are based on choosing a `request_type` of `view_access_key_list` which in turn requires `account_id` as a parameter.  
+
+2. As of [nearcore/2127](https://github.com/nearprotocol/nearcore/pull/2127) we also need to specify finality as one of `optimistic`, `near-final` or `final`.  
```

Comment:
Given the above clarification, we seem need to refrase this comment to indicate that one **can** specify faniality instead of an exact block ID.

Reply:
✅